core/src/main/java/org/polypheny/db/algebra/core/lpg/LpgScan.java

```diff
@@ -40,7 +40,9 @@ public LpgScan( AlgOptCluster cluster, AlgTraitSet traitSet, E graph ) {
 
     @Override
     public String algCompareString() {
-        return "$" + getClass().getSimpleName() + "$" + entity.id + "$" + entity.id;
+        return getClass().getSimpleName() + "$"
+                + entity.id + "$"
```

Comment:
Entity is added twice. Intentional. However, should not make a difference, though.

Reply:
- one entity.